src/test/java/org/cactoos/list/NaturalNumbersTest.java

```diff
@@ -56,4 +56,25 @@ public void notStartsWithNegativeNumbers() {
             new ScalarHasValue<>(0L)
         );
     }
+
+    @Test
+    public void containsNaturalNumbersRange() {
+        MatcherAssert.assertThat(
+            "Can't get range of natural numbers",
+            // @checkstyle MagicNumber (2 lines)
```

Comment:
This string is repeated three times in this class. Perhaps this should be moved to class javadoc? Like

> checkstyle JavadocMethodCheck (500 lines)